packages/presets/src/ai/provider.ts

```diff
@@ -84,6 +85,8 @@ export class AIProvider {
     engine: BlockSuitePresets.AIPhotoEngineService
   ): void;
 
+  static provide(id: 'onboarding', fn: (value: boolean) => void): void;
```

Comment:
Why don't we add `onboarding` to `AIActions`?

Reply:
@pengx17 @Flrande

Reply:
agreed